### ISSUE TITLE
Wired little "bug" made us all laugh 

### ISSUE THREAD

**Issue by Dijkstra1-zz** [McJty/RFToolsDimensions#119]
![](https://cdn.discordapp.com/attachments/315675935053709312/334013554262933504/image.png)

**Comment by McJty** [comment]
yes, know 'issue'. Not sure if I really want to fix it though. It is pretty annoying to fix actually

**Comment by Tahakk** [comment]
You can take it as:

**_It's not a bug, it's a feature_**

Might be late for party, but I couldn't resist
